CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # vscode-powershell Release History
 
+- [vscode-PowerShell #????](https://github.com/PowerShell/vscode-PowerShell/pull/????) -
```

Comment:
This will cause merge conflicts later on if we leave it in. If you put your summary from here into a commit message, it will make its way into the changelog.

Just stick the summary into a commit taking out the changelog entry is probably the easiest way to go. Otherwise there is also the `git commit --amend` -> `git push origin +branch` option.